website/docs/docs/supported-data-platforms.md

```diff
@@ -3,81 +3,102 @@ title: "Supported Data Platforms"
+| Postgres          | [dbt-postgres](https://github.com/dbt-labs/dbt-postgres)       | 1.2.0                    | :white_check_mark: | [Postgres Profile](postgres-profile)     |                                                              | [#db-postgres](https://getdbt.slack.com/archives/C0172G2E273)             | dbt Labs      |
+| Redshift          | [dbt-redshift](https://github.com/dbt-labs/dbt-redshift)       | 1.2.0                    | :white_check_mark: | [Redshift Profile](redshift-profile)     | [Redshift Configs](redshift-configs)                         | [#db-redshift](https://getdbt.slack.com/archives/C01DRQ178LQ)             | dbt Labs      |
+| Snowflake         | [dbt-snowflake](https://github.com/dbt-labs/dbt-snowflake)     | 1.2.0                    | :white_check_mark: | [Snowflake Profile](snowflake-profile)   | [Snowflake Configs](snowflake-configs)                       | [#db-snowflake](https://getdbt.slack.com/archives/C01DRQ178LQ)            | dbt Labs      |
+| Apache Spark      | [dbt-spark](https://github.com/dbt-labs/dbt-spark)             | 1.2.0                    | :white_check_mark: | [Spark Profile](spark-profile)           | [Spark Configs](spark-configs)                               | [#db-databricks-and-spark](https://getdbt.slack.com/archives/C01DRQ178LQ) | dbt Labs      |
+| Starburst & Trino | [dbt-trino](https://github.com/starburstdata/dbt-trino)        | 1.2.0 :construction:     |                    | [Trino Profile](trino-profile)           |                                                              | [#db-presto-trino](https://getdbt.slack.com/archives/C013MLFR7BQ)         | Starburst     |
+
+### Community Adapters
+
+| Data Platform          | Adapter Repository                                                                              | Profile Setup                              | Configuration                              | Slack Channel                                                     | Maintained By |
+| ---------------------- | ----------------------------------------------------------------------------------------------- | ------------------------------------------ | ------------------------------------------ | ----------------------------------------------------------------- | ------------- |
+| Athena                 | [dbt-athena](https://github.com/Tomme/dbt-athena)                                               | [Athena Profile](athena-profile)           |                                            | [#db-athena](https://getdbt.slack.com/archives/C013MLFR7BQ)       | Community     |
+| ClickHouse             | [dbt-clickhouse](https://github.com/ClickHouse/dbt-clickhouse)                                  | [Clickhouse Profile](clickhouse-profile)   | [Clickhouse Configs](clickhouse-configs)   | [#db-redshift](https://getdbt.slack.com/archives/C01DRQ178LQ)     | Clickhouse    |
```

Comment:
> #db-redshift

Points to Redshift instead of ClickHouse?

Reply:
great catch!
```suggestion
| ClickHouse             | [dbt-clickhouse](https://github.com/ClickHouse/dbt-clickhouse)                                  | [Clickhouse Profile](clickhouse-profile)   | [Clickhouse Configs](clickhouse-configs)   | [#db-clickhouse](https://getdbt.slack.com/archives/C03KVDLMNV6)     | Clickhouse    |
```